widgetastic_manageiq.py

```diff
@@ -1589,3 +1589,19 @@ class chart(TimelinesChart):  # NOQA
     @property
     def is_displayed(self):
         return self.title.text == 'Timelines'
+
+
+class ManageIQForm(View):
+    """Generic form view with common action buttons
+
+    .. code-block:: python
+
+        class PageWithFormView(View):
+            @View.nested
+            class form(ManageIQForm):
+                # widgets
+    """
+    continue_button = Button('Continue')
```

Comment:
This creates widgets that might not exist on the form inheriting it, big downside if we don't want exceptions from trying to access buttons that aren't on the page.